src/overlays/actors/ovl_En_Hit_Tag/z_en_hit_tag.c

```diff
@@ -29,21 +28,66 @@ const ActorInit En_Hit_Tag_InitVars = {
     (ActorFunc)NULL,
 };
 
-// static ColliderCylinderInit sCylinderInit = {
-static ColliderCylinderInit D_80BE21F0 = {
-    { COLTYPE_NONE, AT_NONE, AC_ON | AC_TYPE_PLAYER, OC1_ON | OC1_TYPE_PLAYER, OC2_TYPE_1, COLSHAPE_CYLINDER, },
-    { ELEMTYPE_UNK0, { 0x00000000, 0x00, 0x00 }, { 0xF7CFFFFF, 0x00, 0x00 }, TOUCH_NONE | TOUCH_SFX_NORMAL, BUMP_ON, OCELEM_NONE, },
+static ColliderCylinderInit sCylinderInit = {
+    {
+        COLTYPE_NONE,
+        AT_NONE,
+        AC_ON | AC_TYPE_PLAYER,
+        OC1_ON | OC1_TYPE_PLAYER,
+        OC2_TYPE_1,
+        COLSHAPE_CYLINDER,
+    },
+    {
+        ELEMTYPE_UNK0,
+        { 0x00000000, 0x00, 0x00 },
+        { 0xF7CFFFFF, 0x00, 0x00 },
+        TOUCH_NONE | TOUCH_SFX_NORMAL,
+        BUMP_ON,
+        OCELEM_NONE,
+    },
     { 16, 32, 0, { 0, 0, 0 } },
 };
 
-#endif
+void EnHitTag_Init(Actor* thisx, GlobalContext* globalCtx) {
+    ColliderCylinder* cylinder;
+    EnHitTag* this = THIS;
 
-extern ColliderCylinderInit D_80BE21F0;
+    Actor_SetScale(&this->actor, 1.0f);
+    this->actionFunc = EnHitTag_WaitForHit;
+    cylinder = &this->cylinder;
+    Collider_InitAndSetCylinder(globalCtx, cylinder, &this->actor, &sCylinderInit);
+    Collider_UpdateCylinder(&this->actor, cylinder);
```

Comment:
Is this temp needed for anything other than padding the stack? We expect there to be a pad in the main 4 functions for a GameState -> GlobalContext upcast (ask in Discord if you want more information about this), and almost never do we see a temp used like this, hence my suspicion.

Reply:
It does seem that the only thing it is needed for is padding the stack. How should I deal with this? Does this mean that it should not be used as the argument for Collider_InitAndSetCylinder/Collider_UpdateCylinder and I should use &this->collider instead? I actually had it like that originally but changed it in a later commit.

Reply:
Yeah, just use `&this->cylinder` directly in both function calls. Then change `ColliderCylinder* cylinder` to `s32 pad`.